collector/collector.go

```diff
@@ -158,6 +159,17 @@ func NewPostgresCollector(logger *slog.Logger, excludeDatabases []string, dsn st
 	return p, nil
 }
 
+func CollectionTimeout(s string) Option {
```

Comment:
I really like options having a "with" prefix.
```suggestion
func WithCollectionTimeout(s string) Option {
```

Reply:
done